static/skin/taskbar.js

```diff
@@ -7,7 +7,14 @@ const jq = jQuery.noConflict(true);
 jq(document).ready(() => {
     (function ($) {
         const root = $( `link[type='root']` ).attr("href");
-        const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
+        const urlParams = new URLSearchParams(window.location.search);
+
+        content = window.location.pathname.split(`${root}/`)[1].split('/')[0];
```

Comment:
Declare this variable with `let`. Alternatively you can get rid of both `content` and  `urlParams` auxiliary variables if you use the conditional operator `?`.